collect_app/src/main/java/org/odk/collect/android/tasks/SavePointTask.java

```diff
@@ -70,6 +74,11 @@ protected String doInBackground(Void... params) {
                 // write out xml
                 SaveToDiskTask.exportXmlFile(payload, temp.getAbsolutePath());
```

Comment:
If we're pulling `temp` out of SaveToDiskTask just to pass it back in we should probably consider an alternative.